### PR TITLE
MP-1078. Liquidation filtering impl.

### PR DESCRIPTION
**Pull request by piobab**

### CODE REVIEW

File: contracts/liquidation-filterer/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+
+[dependencies]
+mars-outpost = { path = "../../../outposts/packages/outpost", version = "0.1.0" }
```
Comment (dancreee) [inline contracts/liquidation-filterer/Cargo.toml:24]: not ideal having these dependencies as relative paths? once outpost is public we can fix this?

File: contracts/liquidation-filterer/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+cosmwasm-std = "1.0"
+cw-storage-plus = "0.14"
+schemars = "0.8"
+serde = { version = "1.0.144", default-features = false, features = ["derive"] }
+thiserror = "1.0.33"
+
+[dev-dependencies]
+mars-testing = { version = "0.1.0", path = "../../../outposts/packages/testing" }
```
Comment (dancreee) [inline contracts/liquidation-filterer/Cargo.toml:33]: same here with the path

File: contracts/liquidation-filterer/src/contract.rs [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        } => Ok(execute_update_config(deps, env, info, owner, address_provider)?),
+        ExecuteMsg::LiquidateMany {
+            liquidations,
+        } => execute_liquidate(deps, info, liquidations),
+    }
+}
+
+fn execute_update_config(
+    deps: DepsMut,
+    _env: Env,
```
Comment (dancreee) [inline contracts/liquidation-filterer/src/contract.rs:61]: maybe i'm wrong but thought we were not going to include unused variables

**Review by dancreee — APPROVED** [review]
few minor comments but LGTM

**Review by grod220 — APPROVED** [review]

File: contracts/liquidation-filterer/Cargo.toml (hunk 1 above)
Comment (piobab) [reply contracts/liquidation-filterer/Cargo.toml:24]: Yes, that's the plan

**Pull request merged**